.github/workflows/code_coverage.yml

```diff
@@ -0,0 +1,28 @@
+on: [push]
+
+name: Code Coverage
+
+jobs:
+  tarpaulin-codecov:
+    name: Tarpaulin to codecov.io
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+
+      - name: tarpaulin
+        uses: actions-rs/tarpaulin@v0.1.2
+        with:
+          version: '0.15.0'
+          args: '--exclude-files testutils -- --test-threads 1'
```

Comment:
Ideally we should try to add as many features as we can here. I would add the `all-keys` feature to run a few more tests for the `keys` module, and if we can setup the environment with the docker image we could also run the full end to end electrum tests with the `test-electrum` feature.

Unfortunately most of the `blockchain` code (esplora, compact filters) will remain untested for now, at least until we are done with #102

Reply:
There should be a way to combine the feature specific test runs into one on Codecov. I'll get all the features tested on the CI build first and get it added here too. Will let ya know when it's ready.